src/components/specific/files/folder-selector/FolderSelector.vue

```diff
@@ -131,16 +136,19 @@ export default {
     const currentFolder = ref(null);
     const selectedFolder = ref(null);
 
-    const folders = computed(() =>
+    const displayedFiles = computed(() =>
       handler
         .children(currentFolder.value)
         .filter(
           child =>
-            child.type === FILE_TYPES.FOLDER &&
             !props.files.some(f => child.id === f.id) &&
             (props.project.isAdmin ||
               child.userPermission === FILE_PERMISSIONS.READ_WRITE)
         )
+        .sort((a, b) => {
+          if (a.type === "Folder" && b.type !== "Folder") return -1;
```

Comment:
We could use the `FILE_TYPES` enum here, for consistency.

Reply:
I made some modifications in that way
In FolderSelector I ask 7 times if myVar.type === "Folder", so I created a function to generalize the logic
```
 const isFolderType = file => file.type === FILE_TYPES.FOLDER;
```
Then I can use it in setup() and template 
I pushed this modification in a brand new commit

Reply:
Yes ! This definitely sound like a good idea. :)
I think the best place to put this method is in `src/utils/file-structure.js`.

Reply:
As the method takes a file as argument I also suggest that it is simply called `isFolder`.
It would make the code more concise.

Reply:
got it, changes has been made in a new commit